packages/opentelemetry-types/src/trace/span_context.ts

```diff
@@ -35,6 +35,10 @@ export interface SpanContext {
    * lowercase hex characters corresponding to 64 bits.
    */
   spanId: string;
+  /**
+   * Indicates if the SpanContext was propagated from a remote parent.
```

Comment:
Could you specify the default? I assume the default is `false`

Reply:
It's not set to false anywhere explicitly so it'll be `undefined` by default. Should I just change the comment or add a `isRemote: false` to all new SpanContexts?